### PR TITLE
Remove unused SCSS and use Bootstrap's classes more

### PR DESCRIPTION
**Pull request by XhmikosR**
~~**note that this is untested**~~

### CODE REVIEW

File: src/scss/_custom.scss [hunk 1]
```diff
@@ -26,31 +14,17 @@
...
 
-.full-width {
-  width: 100%;
-}
-
 .has-error {
   color: #f86c6b;
 }
 
-#loginform div.row {
```
Comment (XhmikosR) [inline src/scss/_custom.scss]: I couldn't find a `.row` in the login form

File: src/scss/_custom.scss [hunk 2]
```diff
@@ -1,15 +1,3 @@
-@include media-breakpoint-down(xs) {
```
Comment (XhmikosR) [inline src/scss/_custom.scss]: These 2 seem unused

File: src/scss/_custom.scss [hunk 3]
```diff
@@ -67,7 +41,7 @@
   opacity: 1;
   position: absolute;
   background-color: rgba(0, 0, 0, .7);
-  color: #fff;
+  color: $white;
```
Comment (XhmikosR) [inline src/scss/_custom.scss]: Switched to the variables, but I don't mind reverting these.

File: src/scss/_custom.scss [hunk 4]
```diff
@@ -67,12 +41,10 @@
...
   border-radius: 3px;
-  -webkit-transition: all .1s ease;
   transition: all .1s ease;
   pointer-events: none;
-  -webkit-transform: translate(-50%, 0);
```
Comment (XhmikosR) [inline src/scss/_custom.scss]: The vendor prefixes aren't needed since autoprefixer handles this on build time

**Comment by Mcat12** [comment]
> **note that this is untested**

Please do your own testing before submitting for review. Once you have tested it, then I'd be happy to verify it.

**Comment by XhmikosR** [comment]
You are going to test it, aren't you? Otherwise what's the point of a
review?

**Comment by Mcat12** [comment]
Yes, I will test it, but it is expected that the PR author has first tested the changes they are proposing.

**Comment by XhmikosR** [comment]
I can test it again, because now it's been so long I don't even remember.
That being said, more eyes is always better anyway

File: src/components/log/QueryLog.tsx [hunk 5]
```diff
@@ -525,7 +525,7 @@ const columns = (t: TFunction) => [
         return (
           <button
             type="button"
-            className="btn btn-success full-width"
+            className="btn btn-success w-100"
```
Comment (XhmikosR) [inline src/components/log/QueryLog.tsx]: Maybe this and then next ones should be `btn-block instead of `w-100` only?

**Comment by XhmikosR** [comment]
Just tested this and LGTM. I just have one comment about `btn-block`

File: src/components/log/QueryLog.tsx (hunk 5 above)
Comment (Mcat12) [reply src/components/log/QueryLog.tsx]: That change looks fine.

Comment (XhmikosR) [reply src/components/log/QueryLog.tsx]: Done

**Comment by XhmikosR** [comment]
Needs another review for the bnt-block change and then a squash and merge

**Comment by XhmikosR** [comment]
Can't merge this without 2 reviews

**Comment by Mcat12** [comment]
We don't require more than 1 review for dev.

**Pull request merged**